src/solvers/flattening/boolbv_mult.cpp

```diff
@@ -12,6 +12,8 @@ Author: Daniel Kroening, kroening@kroening.com
 
 bvt boolbvt::convert_mult(const exprt &expr)
 {
+  PRECONDITION(expr.id() == ID_mult || expr.id() == "no-overflow-mult");
```

Comment:
Is `"no-overflow-mult"` still used? It does not seem to be assigned anywhere. If not, we can remove it and make `convert_mult` take a `mult_exprt` instead.